CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+
+### Type Labels 
+
+Type labels are labels the define the nature of the issue and/or the correction
+itself.
+
+| Type | Label | Usage |
+|------|------------|----------------------------------------------------------------------------|
+| type | editorial | The issue only involves wording and not normative content. |
+| type | content | The issue involves normative content; resolution requires group consensus. |
+| type | correction | The issue is fixing a recognized problem in the current version. |
+| type | formatting | The issue involves fixing formatting. |
+| type | figure | The issue involves a figure that it missing or needs to be revised. |
+| type | admin | The issue is administrative and NOT about the deliverable. |
+
+### Status Labels 
+
+Status labels are labels that are used to help identify the current state of the
+issue, so that we may accurately classify the work to do on it.
+
+| Status | Label | Usage |
+|--------|-------------------|-------------------------------------------------------------------------------------------------------------------|
+| status | unassigned | The issue is new and has not yet been assigned to anyone. |
+| status | in-progress | The issue has been assigned and work is in progress. |
+| status | needs-review | A resolution (or concrete step forward) has been proposed and needs review. |
+| status | blocked | Progress is currently blocked; the block should be explained in a comment. |
+| status | on-hold | Progress is currently on hold; the reason should be explained in a comment. |
+| status | deferred | Consensus has been reached that this issue can be deferred to a subsequent version. |
+| status | abandoned | Consensus has been reached that this issue can be abandoned. |
+| status | PR-needed | Consensus has been reached and this issue is now waiting for a PR to be submitted. |
+| status | PR-in-progress | The issue is linked to a PR that is in progress |
+| status | PR-completed | The issue is linked to a PR that is complete and waiting for review. |
+| status | PR-accepted | The issue is linked to a PR that has been accepted and is waiting for merge. |
+| status | PR-merged | The issue is linked to a PR that has been merged; this issue can now be closed. |
+| status | status: last-call | The issue has been resolved by some other mechanism documented in the comments and is now in **5 day last call.** |
+
+## Changes to document
+
+- Change proposals can be made by anyone by creating an issue in Git and
+  proposing a pull request against the main branch, containing the proposed
+  changes.
+- Change proposals are made as according to the process below
+- Editors may make changes to the main branch without creating an issue, in the
+  case where an issue has not been already created. If an issue already exists
+  for the change, the editor should follow the standard review process and refer
+  the PR in the issue
+
+### Editing Git process
```

Comment:
Do we want to allow a branch directly off the repo? What about forking for changes?

Reply:
PR's from forking should be allowed IMO for occasional contributors. Branching should be allowed for active contributors/editors. However, I would love to have as simple editing workflow as possible for active contributors. 

We should discuss more on this.
I think our goal for Git-based document management should be:
- Simple and concise way for non-git-savvy contributors to contribute changes (editor, minimal git-understanding required)
- Have Issue & PR -based workflow for all changes
- Enable editors with tools and workflows that contribute positively (i.e. help manage changes) and not negatively (too complex editing).